### PR TITLE
Homework Wordpress Kristina

### PR DESCRIPTION
**Pull request by kristinatomic**

### CODE REVIEW

**Comment by markomitranic** [comment]
Unfortunately your homework is the only one i have not tested.

You had some issues with exporting the SQL file and we should work on that in the future. The file you uploaded was a zipped folder full of sql files, instead of a single SQL dump.

The same goes for the files themselves, for some reason you decided to zip them, which beats the purpose of using git at all.

We will work on this tomorrow and try to fix these issues.

File: wp-content/themes/alumni/style.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Theme Name: Alumni
+Theme URI: http://wordpress.org/themes/twentythirteen
+Author: Kristina Tomic
+Author URI: http://wordpress.org/
+Description: The 2013 theme for WordPress takes us back to the blog, featuring a full range of post formats, each displayed beautifully in their own unique way. Design details abound, starting with a vibrant color scheme and matching header images, beautiful typography and icons, and a flexible layout that looks great on any device, big or small.
+Version: 1.0
+License: GNU General Public License v2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+Tags: black, brown, orange, tan, white, yellow, light, one-column, two-columns, right-sidebar, flexible-width, custom-header, custom-menu, editor-style, featured-images, microformats, post-formats, rtl-language-support, sticky-post, translation-ready
+Text Domain: twentythirteen
```
Comment (markomitranic) [inline wp-content/themes/alumni/style.css]: Change the text domain to a short version of theme name. For example if theme was named "My Awesome Theme" the text domain would be `my-awesome-theme`. So in your case it is `alumni`. 

For now we do not yet use this information anywhere, but it is useful later on, as your wordpress will use this information when translating themes for example. Its good to remember.

File: wp-content/themes/alumni/index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (markomitranic) [inline wp-content/themes/alumni/index.html]: Index.html inside this theme is an unused file. It should not contain anything.

In fact, if you look around, you will notice that wordpress decided that every folder MUST have at least one empty file with a single line of commented out text `// Silence is golden.`. There is a pretty cool reason behind that, and i will explain that on the lecture.

For now, just empty the contents of this file, as it is never used for anything. And google around, try to find out why they decided to have a convention with these empty html index files with that particular comment line.

Your task will be to explain that to us in class :) and i will help you with that. Ping me on Slack if you have any trouble finding or understanding that information.

File: wp-content/themes/alumni/header.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+<html>
+<head>
+	<title></title>
+</head>
+<body>
+
+<nav>
+	<ul>
+		<li></li>
+		<li></li>
```
Comment (markomitranic) [inline wp-content/themes/alumni/header.php]: Empty `<li>` tags are not valid HTML.

In general, git hygiene rules are that you should never, ever commit unfinished or uncomplete or unused code. When you start working in production, you will often find yourself making a quick glance at the files before every commit, just to make sure that you did not leave any comments or invalid or unused code there.

So you should either put some text inside (do that), or remove it enterely.

I don't remember now, but if i made some code like this in class, i should be hanged 🗡 🗡 🗡  if i do, please feel free to remind me and warn me about it in class.

File: wp-content/themes/Assignment1/footer.php [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<ul>
+			<li><a href="#">Facebook</a></li>
+			<li><a href="#">Instagram</a></li>
```
Comment (markomitranic) [inline wp-content/themes/Assignment1/footer.php]: Please fix indentation here, as you can see the `<ul>` tag is indented bad compared to it's closing tag.

File: wp-content/themes/Assignment1/header.php [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>
+		<?php 
+		bloginfo('name');
+		bloginfo('description');
```
Comment (markomitranic) [inline wp-content/themes/Assignment1/header.php]: Good thinking and close enough. But this won't do. 🥇 

The task was to try and create output of the title like this `Title ~ Description` and your solution will output this `TitleDescription`. Please think about how you can fix this, it is a simple solution really and you really are close. Ping me on Slack if you can't think of anything!

General advice: When you get a certain task, always double check that your solution works as planned. As Robert Martin wrote in his book, "QA should find nothing". It basically says that you should always double check that you did in fact do what was asked. It is a really common programmer mistake to not double check themselves.

File: wp-content/themes/Assignment1/index.php [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+	<main>
+		<h1>Page Title</h1>
+		<div class="content">
+			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Explicabo nemo ea quibusdam quo quasi sunt neque modi hic quisquam necessitatibus in expedita officia a iure, nihil dolor harum amet eius.</p>
+			<p>Exercitationem perspiciatis unde, incidunt id delectus, quidem maiores quia, laudantium maxime repellat nobis itaque ipsum dolores officiis. Eos ut accusamus fugit voluptas facere deserunt, ipsam ducimus et delectus, eligendi odit.</p>
+		</div>
+	</main>
+
+
```
Comment (markomitranic) [inline wp-content/themes/Assignment1/index.php]: This is the only place in your code where you did type 2x enter. Why?

Change Request: Please remove 1 enter so there is only one left like everywhere else.

General Advice: When programming, always pick a coding style, and stick to it. So if you say ok im gonna use 2x enter between code blocks everywhere, feel free to do that! But if your coding style varies, the code gets harder to read, and you will agree it is a really stupid problem and a really easy fix. 👍

File: wp-content/themes/alumni/style.css [hunk 7]
```diff
@@ -12,4 +12,6 @@ Text Domain: twentythirteen
 
 This theme, like WordPress, is licensed under the GPL.
 Use it to make something cool, have fun, and share what you've learned with others.
-*/
\ No newline at end of file
+*/
+
+@import url("../twentyseventeen/style.css");
```
Comment (markomitranic) [inline wp-content/themes/alumni/style.css]: Not entirely sure what did you try here?

Don't get me wrong, trying out things is highly encouraged, so whatever it is, good job :P
But i am geuinely interested, so if you find the time, please respond here or on slack, thanks

Comment (kristinatomic) [reply wp-content/themes/alumni/style.css]: Well, here, I have tried to do some magic :D 
I wanted to transfer all css from twentyseventeen to alumni and make it all functional, but after I realize this is not how WordPress works and I have to type code manually , I guess we will learn that in upcoming classes. :)

File: wp-content/themes/Assignment1/index.php (hunk 6 above)
Comment (kristinatomic) [reply wp-content/themes/Assignment1/index.php]: Understood :)

**Pull request closed without merging**